examples/example_batch_execution.py

```diff
@@ -0,0 +1,104 @@
+# -----------------------------------------------------------
+# This example presents the code for the batch/online execution
+# of the pipeline.
+# Additionally, the regressor in this pipeline is retrained
+# if the datapoint corresponds to midnight.
+# -----------------------------------------------------------
+
+import pandas as pd
+from sklearn.preprocessing import StandardScaler
+from sklearn.svm import SVR
+
+# Import the pyWATTS pipeline and the required modules
+from pywatts.callbacks import CSVCallback, LinePlotCallback
+from pywatts.core.computation_mode import ComputationMode
+from pywatts.core.pipeline import Pipeline
+from pywatts.modules import ClockShift, LinearInterpolater, RollingRMSE, SKLearnWrapper
+
+
+# This function creates and returns the preprocessing pipeline
+def create_preprocessing_pipeline(power_scaler):
+    pipeline = Pipeline(path="../results/preprocessing")
+
+    # Deal with missing values through linear interpolation
+    imputer_power_statistics = LinearInterpolater(method="nearest", dim="time",
+                                                  name="imputer_power")(x=pipeline["scaler_power"])
+    # Scale the data using a standard SKLearn scaler
+    scale_power_statistics = power_scaler(x=imputer_power_statistics)
+
+    # Create lagged time series to later be used in the regression
+    ClockShift(lag=1)(x=scale_power_statistics)
+    ClockShift(lag=2)(x=scale_power_statistics)
```

Comment:
`UserWarning: Naming Conflict: ClockShift is renamed to. ClockShift_1`
could be avoided by specifying names.
Then, adaptions are required in lines
- [ ] 42
- [ ] 43
- [ ] 48
- [ ] 49
- [ ] 77
- [ ] 78
- [ ] 95
- [ ] 96